apps/connect-javascript/src/portabl.js

```diff
@@ -0,0 +1,46 @@
+const PORTABL_ACCOUNT_ID = window._env_.JS_APP_PORTABL_ACCOUNT_ID;
+const PORTABL_CONNECT_DOMAIN = window._env_.JS_APP_PORTABL_CONNECT_DOMAIN;
+const PORTABL_WALLET_DOMAIN = window._env_.JS_APP_PORTABL_WALLET_DOMAIN;
+
+const { ConnectClient } = Portabl.connect;
+const connectClient = new ConnectClient({
+  accountId: PORTABL_ACCOUNT_ID,
+  connectDomain: PORTABL_CONNECT_DOMAIN,
+  walletDomain: PORTABL_WALLET_DOMAIN,
+});
+
+const connectForm = document.querySelector('.connect-login');
+connectForm.onsubmit = (e) => {
+  e.preventDefault();
+  connectClient.loginWithRedirect();
+};
+
+const logoutBtn = document.querySelector('.logout-btn');
+logoutBtn.onclick = (e) => {
+  e.preventDefault();
+  connectClient.logout();
+  updateUI();
+};
+
+window.onload = async () => {
+  await connectClient.handleRedirectCallback();
```

Comment:
Check if there is response code in the url before calling handleRedirectCallback